### PR TITLE
Add users step

### PR DESCRIPTION
**Pull request by zemberdotnet**
# Description

Adds `tanium_user` and `tanium_account_has_user` relationship.

### CODE REVIEW

File: docs/jupiterone.md [hunk 1]
```diff
@@ -107,6 +108,7 @@ The following relationships are created:
 | Source Entity `_type` | Relationship `_class` | Target Entity `_type` |
 | --------------------- | --------------------- | --------------------- |
 | `tanium_account`      | **HAS**               | `tanium_endpoint`     |
+| `tanium_account`      | **HAS**               | `tanium_endpoint`     |
```
Comment (VDubber) [inline docs/jupiterone.md:111]: This doesn't seem right. Wasn't this a bug that Austin fixed?

Comment (zemberdotnet) [reply docs/jupiterone.md]: Actually my bad. Fixed the step metadata!

File: src/tanium/rest-types.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  domain: string;
+  display_name: string;
+  group_id: number;
+  effective_group_id: number;
+  deleted_flag: boolean;
+  locked_out: number;
+  creation_time: Date;
+  modification_time: Date;
+  last_login: Date;
+  previous_login: Date;
```
Comment (VDubber) [inline src/tanium/rest-types.ts:14]: last vs previous, what's the difference?

File: src/tanium/rest-types.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  permissions: string[];
+  effective_content_set_privileges: EffectiveContentSetPrivilege[];
+  local_admin_flag: number;
+  metadata: any[];
+  content_set_roles: ContentSetRole[];
+  roles: any[];
```
Comment (VDubber) [inline src/tanium/rest-types.ts:20]: Future relationship?

File: src/tanium/rest-types.ts (hunk 2 above)
Comment (zemberdotnet) [reply src/tanium/rest-types.ts:14]: `previous_login` is the login before `last_login`

**Review by VDubber — APPROVED** [review]

**Pull request merged**